patch_bpf.py

```
@@ -0,0 +1,51 @@
+import re
+
+with open("crates/logfwd-ebpf-proto/sensor-ebpf/sensor-ebpf-kern/src/main.rs", "r") as f:
```

Ruff (0.15.11)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)